### PR TITLE
Ensure newly created vaults are unlocked

### PR DESCRIPTION
**Pull request by Gudahtt**
A new vault should be unlocked after being created, because the password is submitted as part of creating the vault. This was accidentally broken as part of #148. A unit test has been added to ensure this doesn't happen again.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ describe('KeyringController', function () {
   });
 
   describe('createNewVaultAndKeychain', function () {
-    it('should set a vault on the configManager', async function () {
+    it('should create a new vault', async function () {
```
Comment (Gudahtt) [inline test/index.js:88]: This change is unrelated; I just happened to notice a reference to a now-extinct "configManager" abstraction in another test. This was the last remaining reference to it.

**Review by darkwing — APPROVED** [review]
